examples/existing-network/README.md

```diff
@@ -0,0 +1,63 @@
+# EXAMPLE: Standalone / Active-Active, External Services Installation of Terraform Enterprise
```

Comment:
@sandrampeter , just realized that this should also say that it's using an existing network. Can you create a new PR to fix that? Also, if the node count is variable, then we can leave off the `Standalone / Active-Active` because it's confusing. 
```
# EXAMPLE: External Services Installation of Terraform Enterprise into an Existing Network
```